lib/rubocop/cop/rspec/example_wording.rb

```diff
@@ -33,8 +33,8 @@ class ExampleWording < Cop
         MSG_SHOULD = 'Do not use should when describing your tests.'.freeze
         MSG_IT     = "Do not repeat 'it' when describing your tests.".freeze
 
-        SHOULD_PREFIX = 'should'.freeze
-        IT_PREFIX     = 'it '.freeze
+        SHOULD_PREFIX = /\Ashould(?:n't|not)?\b/i
```

Comment:
Weirdly, it works because it flags the regular should followed by a boundary character and then the `Wording` class has a correct regex. Technically doing `|not` or `| not` is redundant since it includes `\b`. I think I'll just delete that side of the alternation since it's effectively unreachable code.